src/client/setup/setup.tsx

```diff
@@ -76,39 +84,82 @@ interface SetupMainProps {
  */
 function SetupMain(props: SetupMainProps) {
     const navigate = useNavigate();
+
+    //we have this slider value because it's the only way to automatically
+    //update the slider. Set it to currenet user setting initially.
+    //instead of changing the value, we just refresh the page.
+    const [sliderValue] = useState(getUserSetting());
+
+    //we have this because until its declared false, we know our program is still rendering.
+    //So, we wait for the useEFfect to then know what we're done refreshing
+    const [rendering] = useState("true");
+
+    // This effect will run after every render
+    useEffect(() => {
+        //now that we're done refreshing, we set the status of refreshing to false
+        localStorage.setItem("refreshing", "false");
+    }, [rendering]);
+
     const debugButton = (
         <Button
             minimal
-            style={{ float: "right" }}
+            style={{ float: "right", color: "white" }}
             icon="cog"
             onClick={() => navigate("/debug")}
         />
     );
-
     /** computer, human, and puzzle buttons */
     const actions = (
         <>
             <Button
                 large
-                text="Play with the computer"
+                text="Play With The Computer"
                 rightIcon="arrow-right"
                 intent="primary"
                 onClick={() => props.onPageChange(SetupType.COMPUTER)}
+                className={buttonColor()}
             />
             <Button
                 large
-                text="Play against a human"
+                text="Play Against A Human"
                 rightIcon="arrow-right"
                 intent="primary"
                 onClick={() => props.onPageChange(SetupType.HUMAN)}
+                className={buttonColor()}
             />
             <Button
                 large
                 text="Puzzle"
                 rightIcon="arrow-right"
                 intent="primary"
                 onClick={() => props.onPageChange(SetupType.PUZZLE)}
+                className={buttonColor()}
             />
+            <h3 className={textColor()}>Display Settings:</h3>
+            <Slider
```

Comment:
I think you should put the Button/icon next to the debug icon in the popup instead of having a whole slider here